modules/custom/az_global_footer/az_global_footer.install

```diff
@@ -188,3 +188,36 @@ function az_global_footer_update_1020801() {
 
   return t('Updated %count X-Twitter link(s) & icon(s) in the global footer.', ['%count' => $updated_count]);
 }
+
+/**
+ * Update UAlert footer links.
+ *
+ * Replace UAlert links in the global footer with Annual Security Report.
+ */
+function az_global_footer_update_1021001() {
```

Comment:
If we want to backport this change to 2.9.x, we should probably change this update number.
```suggestion
function az_global_footer_update_1020901() {
```